clients/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class Client(models.Model):
+    name = models.CharField(max_length=250)
+    address = models.CharField()
+    contact_number = models.CharField(max_length=20)
+    is_active = models.BooleanField()
```

Comment:
For `is_active` and any other audit tracking fields, lets put them in abstract models that serves as base classes for our main models.

Reply:
Addressed here: https://github.com/jino-d/warehouse/pull/8/files